app/controllers/users/registrations_controller.rb

```diff
@@ -52,7 +52,14 @@ def edit
   end
 
   def update
-    if @user.update  configure_registration_update_parameters
+    if configure_registration_update_parameters[:email] != @user.email
+      @newemail = configure_registration_update_parameters[:email]
+      UserMailer.with(user: @user).update_email.deliver_now
+      UserMailer.with(user: @user, email: @newemail).new_email.deliver_now
```

Comment:
I think this part looks good but I also like the idea we talked about with adding an attribute onto the user for a previous email.